src/Gui/Dialogs/DlgAddPropertyVarSet.cpp

```diff
@@ -667,6 +729,24 @@ void DlgAddPropertyVarSet::changeEvent(QEvent* e)
     QDialog::changeEvent(e);
 }
 
+void DlgAddPropertyVarSet::valueChangedEnum()
+{
+    auto* propEnum =
+        static_cast<App::PropertyEnumeration*>(propertyItem->getFirstProperty());
```

Comment:
Another potential improvement - `getFirstProperty` could be templated with such static_cast in place